source/lac/lapack_full_matrix.cc

```diff
@@ -130,10 +130,11 @@ LAPACKFullMatrix<number>::operator = (const double d)
 
 
 template <typename number>
+template <typename number2>
 void
 LAPACKFullMatrix<number>::vmult (
-  Vector<number>       &w,
-  const Vector<number> &v,
+  Vector<number2>       &w,
+  const Vector<number2> &v,
```

Comment:
How do you support the case `number != number2`? The function calls either `dgemv` or `dgemv`, but there is no mixed precision analog.